src/components/AboutUsHeader/styles.module.css

```diff
@@ -0,0 +1,94 @@
+/**
+ * CSS files with the .module.css suffix will be treated as CSS modules
+ * and scoped locally.
+ */
+
+
+.about_header {
+  position: relative;
+  overflow: hidden;
+  height: 45vh;
+}
+      
+.container {
+  width: 100vw;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/static/img/main_bg.png");
+  background-position: left center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  gap: 2rem;
+}
+      
+.svg_container {
+  width: 20vw;
+  height: 60%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+      
+.svg {
+  width: 75%;
+  height: 75%;
+}
+      
+.information_container {
+  width: 40vw;
+  height: 60%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+      
+.title {
+  font-family: "Manrope", sans-serif;
+  font-weight: 400;
+  font-size: 4.5vw;
+  line-height: 73%;
+}
+      
+.subtitle {
+  font-family: "Manrope", sans-serif;
+  font-weight: 400;
+  font-size: 1.25vw;
+  line-height: 117%;
+  margin-top: 1.5rem;
+}
+      
+@media screen and (max-width: 996px) {
+  .about_header {
+    height: auto;
+  }
+
+  .title {
+    font-size: 4vw;
+  }
+
+  .subtitle {
+    font-size: 2vw;
+    margin-top: -1rem;
+  }
+
+  .container {
+    height: auto;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+    padding: 2rem 0;
+  }
+
+  .information_container {
+    height: 20vw;
+    justify-content: space-evenly;
+    align-items: center;
+    text-align: center;
+  }
+}
+
+
+    
```

Comment:
Please create a new PR and remove too many new lines

Reply:
@Siegfriedk noted! creating new one right away

Reply:
Due to conflicts @Siegfriedk I see now why a complete new PR is requested :/